tests/FSharpPlus.Tests/General.fs

```diff
@@ -1544,10 +1544,18 @@ module Applicative =
         let run (ZipList x) = x
         let run' (ZipList' x) = x
 
+
         // Test Applicative (functions)
+        let fn = 
+            let mutable x = 0
+            fun () -> 
+                x <- x + 1
+                x
         let res607 = map (+) ( (*) 100 ) 6 7
         let res606 = ( (+) <*>  (*) 100 ) 6
         let res508 = (map (+) ((+) 3 ) <*> (*) 100) 5
+        let res123 = (map tuple3 fn <*> fn <*> fn) ()
+        let res456 = (sequence [fn; fn; fn] : unit -> int list) ()
```

Comment:
I'm not sure if it's appropriate to include an integration-style test via `sequence` here in the test module for Applicative.  I could move it to a test module for Traversable, or just get rid of it, if either option is better than this.

Reply:
You can leave it here, it's ok since we're using the traversable to test the applicative.